.gitattributes

```diff
@@ -8,3 +8,5 @@
 .travis.yml export-ignore
 CHANGELOG.md export-ignore
 phpunit.xml.dist export-ignore
+phpstan-baseline.neon export-ignore
+phpstan.neon.dist export-ignore
```

Comment:
So why two files? Let's leave only one?

Reply:
One is the phpstan config and the other one is generated by phpstan baseline command.

Those cannot be combined